tests/components/hand-controls.test.js

```diff
@@ -41,6 +41,23 @@ suite('hand-controls', function () {
       trackedControls = el.components['tracked-controls'];
       trackedControls.controller = {id: 'Foobar', connected: true};
 
+      component.pressedButtons['grip'] = true;
+      component.pressedButtons['trigger'] = false;
+      component.pressedButtons['trackpad'] = true;
+      component.pressedButtons['thumbstick'] = false;
+      component.pressedButtons['menu'] = false;
+      component.pressedButtons['AorX'] = false;
+      component.pressedButtons['BorY'] = false;
+      component.pressedButtons['surface'] = false;
+      assert.equal(component.determineGesture(), 'Point');
+    });
+
+    test('makes point gesture on vive', function () {
+      var trackedControls;
+      el.setAttribute('tracked-controls', '');
+      trackedControls = el.components['tracked-controls'];
+      trackedControls.controller = {id: 'OpenVR Vive', connected: true};
```

Comment:
Is `OpenVR Vive` really how the controllers are appearing these days?

Reply:
In Firefox is `OpenVR Gamapad`

Reply:
Makes sense to use a real value, I'll update to that.